src/utils/html.rs

```diff
@@ -72,3 +72,38 @@ fn extract_class(node: &Handle) -> String {
         _ => String::new()
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+
+    fn round_trip(a: &str) -> String {
+        let parser = parse_document(RcDom::default(), ParseOpts::default());
+        stringify(parser.one(a).document)
+    }
+    fn mostly_equal(a: &str, b: &str) -> bool {
+        round_trip(a.trim()).trim() == round_trip(b.trim()).trim()
+    }
```

Comment:
I would rather not depend on the parser from the tests directly, just exactly compare, is there a reason you chose to do this?

Reply:
The whitespace was messing up the tests. Also if we compare exactly then it's dependent on how it's serialized by the parser, which might change in a minor version. The way the document is parsed is much less likely to change between version or between parsers.

Reply:
I would rather update our tests than have this, even if that means updating on a minor version bump.